radionets/dl_framework/learner.py

```diff
@@ -63,20 +65,31 @@ def define_learner(
                 CudaCallback,
             ]
         )
-    if not test:
+    if train_conf["comet_ml"] and not lr_find and not plot_loss:
         cbfs.extend(
             [
-                SaveTempCallback(model_path=model_path),
-                AvgLossCallback,
-                DataAug,
+                CometCallback(
+                    name=train_conf["project_name"],
+                    test_data=train_conf["data_path"],
+                    plot_n_epochs=train_conf["plot_n_epochs"],
+                    scale=train_conf["scale"],
+                ),
             ]
         )
-    if train_conf["telegram_logger"] and not lr_find:
+    if not test:
         cbfs.extend(
             [
-                TelegramLoggerCallback(model_name=model_name),
+                SaveTempCallback(model_path=model_path),
+                AvgLossCallback,
+                DataAug,
             ]
         )
+        if train_conf["telegram_logger"] and not lr_find:
+            cbfs.extend(
+                [
+                    TelegramLoggerCallback(model_name=model_name),
+                ]
+            )
```

Comment:
Maybe we can think about deleting the Telegram Logger, since comet_ml provides more and better functionalities.

Reply:
we could delete it, but I don't mind keeping it :D

Reply:
Ok then we will keep it for now :D